Makefile

```diff
@@ -123,6 +123,7 @@ test: all
 	$(MAKE) cctest
 	$(PYTHON) tools/test.py --mode=release -J \
 		addons doctool inspector known_issues message pseudo-tty parallel sequential
+	$(MAKE) doclint
```

Comment:
The thing is, there is a project named `mdlint`, and that's not the one we are using.